src/API/NoteActions.php

```diff
@@ -115,6 +119,8 @@ public function trigger_note_action( $request ) {
 			$tracks_event = 'wcadmin_inbox_action_click';
 		}
 
+		$screen_name = $request->get_param( 'screen' );
```

Comment:
Could we not grab the referer from the request in here instead of manually sending it from the client?

Reply:
Hi @joshuatf, thanks for the review.
I chose to send the param to try to have only one method where the screen names come from, but the solution you mentioned (take the referer from the request) is better. I applied it in the commits 8af0e02 and 3ff440f
In this PR I also moved the method `getScreenName` to a more general `utils` file. Now it is not directly affecting this PR but I'll leave this method there because I think that it's a better place for it.

Reply:
>In this PR I also moved the method getScreenName to a more general utils file. Now it is not directly affecting this PR but I'll leave this method there because I think that it's a better place for it.

Nice clean up!  Agreed, good change here.